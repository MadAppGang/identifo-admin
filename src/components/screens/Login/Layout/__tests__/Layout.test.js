import LoginScreenLayout from '..';

describe('<LoginScreenLayout />', () => {
  test('renders as expected', () => {
    expect(shallow(<LoginScreenLayout />)).toMatchSnapshot();
  });

  test('renders children when passed in', () => {
    const component = shallow(
      <LoginScreenLayout>
        <div>Hello, world!</div>
      </LoginScreenLayout>,
    );

    expect(component.contains(<div>Hello, world!</div>)).toBe(true);
  });
});
