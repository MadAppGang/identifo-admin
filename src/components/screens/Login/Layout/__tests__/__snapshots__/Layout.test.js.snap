// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginScreenLayout /> renders as expected 1`] = `
<div
  className="iap-login-layout"
/>
`;
